### PR TITLE
docs: add dropbox to users list

### PR DESCRIPTION
**Pull request by exemplary-citizen**
### SUMMARY
Add Dropbox to users list

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

cc: @bkyryliuk

### CODE REVIEW

File: RESOURCES/INTHEWILD.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ Join our growing community!
 - [Symmetrics](https://www.symmetrics.fyi)
 
 ### Others
+ - [Dropbox](https://www.dropbox.com/)
```
Comment (exemplary-citizen) [inline RESOURCES/INTHEWILD.md:145]:
```suggestion
 - [Dropbox](https://www.dropbox.com/) [@bkyryliuk]
```

Comment (exemplary-citizen) [reply RESOURCES/INTHEWILD.md:145]:
```suggestion
 - [Dropbox](https://github.com/dropbox)
```

**Review by exemplary-citizen — COMMENTED** [review]
added some suggestions. lmk what you like best

File: RESOURCES/INTHEWILD.md (hunk 1 above)
Comment (villebro) [reply RESOURCES/INTHEWILD.md:145]: I think the first one " - [Dropbox](https://www.dropbox.com/) [@bkyryliuk]" makes most sense.

Comment (bkyryliuk) [reply RESOURCES/INTHEWILD.md:145]: +1

**Review by bkyryliuk — APPROVED** [review]

**Pull request merged**